packages/transition-backend/file/definitions/definitions-en.tex

```diff
@@ -605,782 +590,771 @@ \subsection*{Temps et durées • \textit{Time and durations}}
+  \makecell[r]{Peak-hour coefficient (PHC)} & \[\gamma_{PHC}\] & - & \[\frac{P_{up}}{P_{max}} = \frac{1}{\gamma_{PHF}}\] & Coefficient used to account for the maximum 15-minute volume. \(1 \leq \gamma_{PHC} \leq 4\). \\
+  \hline
+  \label{design_volume}
+  \makecell[r]{Design volume} & \[P_d\] & \[pass/h\] & \[\gamma_{PHC} P_{max} = P_{up}\] & Volume used to determine required line capacities or the capacity utilization coefficient. \\
+  \hline
+  \label{daily_path_volume}
+  \makecell[r]{Daily path volume} & \[{{P_p}_{day}}\] & \[pass/day\] & \[\sum_{i=0}^{23} {P_i}\] where \(P_i\) represents passenger volume between hour \(i\) and hour \(i+1\) on the path & Number of passengers per day transported on the path, in one direction. \\
+  \hline
+  \label{daily_line_volume}
+  \makecell[r]{Daily line volume} & \[{{P_L}_{day}}\] & \[pass/day\] & \[\sum_{i=0}^{23} {P_i}\] where \(P_i\) represents passenger volume between hour \(i\) and hour \(i+1\) & Number of passengers per day transported on all trips of the line. \\
+  \hline
+  \label{potential_demand}
+  \makecell[r]{Potential demand} & \[P_{pot}\] & \[pass/h\] & \[P_{lat} + P_{obs}\] & Passenger volume that would exist if service and pricing were optimal. \\
+  \hline
+  \label{latent_demand}
+  \makecell[r]{Latent demand} & \[P_{lat}\] & \[pass/h\] & \[P_{pot} - P_{obs}\] & Unserved or unsatisfied demand. \\
+  \hline
+  \label{observed_volume}
+  \makecell[r]{Observed volume} & \[P_{obs}\] & \[pass/h\] & \[P_{pot} - P_{lat}\] & Passenger volume observed in the network or line. \\
+  \hline
+  \end{longtable}
 
 
 
 
 \pagebreak
-\subsection*{Capacités, travail et productivité • \textit{Capacities, work and productivity}}
+\subsection*{Capacities, work and productivity}
 
 \begin{longtable}{%
-  R{.25\NetTableWidth}%
-  C{.08\NetTableWidth}%
-  C{.12\NetTableWidth}%
-  C{.15\NetTableWidth}%
-  L{.40\NetTableWidth}%
+    R{.25\NetTableWidth}%
+    C{.08\NetTableWidth}%
+    C{.12\NetTableWidth}%
+    C{.15\NetTableWidth}%
+    L{.40\NetTableWidth}%
 }
 \hline
-\makecell[r]{Définition \\ \textit{Definition}} & \makecell[c]{Symbole \\ \textit{Symbol}} & \makecell[c]{Unité \\ \textit{Unit}} & \makecell[c]{Expression \\ \textit{Expression}} & \makecell[l]{Description \\ \textit{Description}} \\ 
-\hline
+\makecell[r]{Definition} & \makecell[c]{Symbol} & \makecell[c]{Unit} & \makecell[c]{Expression} & \makecell[l]{Description} \\\hline
 \hline
 \endhead
 \label{capacity}
-\makecell[r]{Capacité \\ \textit{Capacity}} & \[C\] & \[places/h\] & - & Capacité d'une ligne, dans une direction, par heure. \\
+\makecell[r]{Capacity} & \[C\] & \[places/h\] & - & Line capacity, in one direction, per hour. \\
 \hline
 \label{unit_capacity}
-\makecell[r]{Capacité de l'unité \\ \textit{Unit capacity}} & \[c_u\] & \[places\] & - & Capacité d'une unité. \\
+\makecell[r]{Unit capacity} & \[C_{u}\] & \[places\] & \[C_{{u}_{se}} + C_{{u}_{st}}\] & Total number of seated and standing places in the unit. \\
```

Comment:
addtion {} around u in C_u
equation quite different that previous, pleause double check